### PR TITLE
Centralise porthandling

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: src/common/pf_eth.c [hunk 1]
```diff
@@ -158,6 +158,23 @@ int pf_eth_init (pnet_t * net, const pnet_cfg_t * p_cfg)
...
 }
 
+int pf_eth_send_on_physical_port (
+   pnet_t * net,
+   int loc_port_num,
+   pnal_buf_t * buf)
+{
+   pf_port_t * p_port_data = pf_port_get_state (net, loc_port_num);
+
+   // TODO Handle tail tagging: Put loc_port_num in p_buffer metadata
```
Comment (olbjo) [inline src/common/pf_eth.c:168]: I think you should use a /*  */ comment

**Review by olbjo — APPROVED** [review]
This works for me. Checkin commit mentions that this is a preparation for tail-tagging which is good.
:+1:

File: src/common/pf_eth.c (hunk 1 above)
Comment (pyhys) [reply src/common/pf_eth.c]: Done

**Review by olbjo — COMMENTED** [review]
Looks good!

**Pull request closed without merging**